Database/Corrections/sodQuestFixes.lua

```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410095,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90206] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{1115,1116,1117,1118,208711}},
```

Comment:
Spawns of `208711` should be cleaned up.

Reply:
Added in: https://github.com/Questie/Questie/pull/5608/commits/f3e8eb78740cd80a1e9962c386390fafc53a7857